README.md

```diff
@@ -48,7 +53,10 @@ add her to the list of guests in the castle.
 
 Write your code here:
 ```ruby
-# code here
+
+town[:resident] - ["Belle"]
```

Comment:
Oops ignore my above comment, you can actually do this. Learn something new every day :)